src/main/js/components/rows/Row.jsx

```diff
@@ -109,14 +107,13 @@ class Row extends React.Component {
       // If value is multi-language just render cell
       if (cell.isMultiLanguage) {
         return <Cell key={idx} cell={cell} langtag={langtag} selected={selected} editing={editing}
-                     shouldFocus={shouldFocus} />;
+                     shouldFocus={shouldFocus} row={self.props.row} table={self.props.table} />;
       }
     })
   };
 
   contextMenuHandler = e => {
-    e.preventDefault();
-    ActionCreator.showRowContextMenu(this.props.row, this.props.langtag, e.pageX, e.pageY, this.props.table);
+    e.stopPropagation();
```

Comment:
Ah okay, misunderstood. Since `renderSingleLanguageCell` seems to use it and each single language cell would create a new handler, I wouldn't inline this. It's also used in two different code parts here, so should be okay.